test/system/liquidation.test.js

```diff
@@ -180,7 +181,7 @@ describeFn("System -- liquidation", () => {
       await expect(parseInt(tx.gasLimit)).to.be.lessThan(518000)
 
       const txReceipt = await ethers.provider.getTransactionReceipt(tx.hash)
-      await expect(parseInt(txReceipt.gasUsed)).to.be.lessThan(255000)
+      await expect(parseInt(txReceipt.gasUsed)).to.be.lessThan(400000)
```

Comment:
I am surprised by this increase. Can we check on a local geth instance what is the cost? It is possible that the Hardhat does not properly refund.